library/ssl_tls13_server.c

```diff
@@ -1613,10 +1613,12 @@ static int ssl_tls13_parse_client_hello(mbedtls_ssl_context *ssl,
             case MBEDTLS_TLS_EXT_RECORD_SIZE_LIMIT:
                 MBEDTLS_SSL_DEBUG_MSG(3, ("found record_size_limit extension"));
 
-                ret = mbedtls_ssl_tls13_parse_record_size_limit_ext(ssl, p, extension_data_end);
+                ret = mbedtls_ssl_tls13_parse_record_size_limit_ext(
+                    ssl, p, extension_data_end);
 
-                /* TODO: Return unconditionally here until we handle the record size limit correctly.
-                 *            Once handled correctly, only return in case of errors. */
+                /* TODO: Return unconditionally here until we handle the record
+                 * size limit correctly.
+                 * Once handled correctly, only return in case of errors. */
```

Comment:
```suggestion
                /* 
                 * TODO: Return unconditionally here until we handle the record
                 *       size limit correctly. Once handled correctly, only 
                 *       return in case of errors. 
                 */
```